src/standalone/Soil/Biogeochemistry/co2_parameterizations.jl

```diff
@@ -42,7 +42,7 @@ which is related to the total soil porosity (`ν`) and
 volumetric soil water content (`θ_w = θ_l+θ_i`).
 """
 function volumetric_air_content(θ_w::FT, ν::FT) where {FT}
-    θ_a = ν - θ_w
+    θ_a = max(ν - θ_w, eps(FT))
```

Comment:
it's possible this could happen and be OK, for example, if saturated soil freezes, theta_w > \nu because ice has a bigger volume than water does?